src/components/inputs/buttons/CopyTextButton/CopyTextButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+
+import { CopyIcon } from '../../../icons';
+
+function CopyTextButton({
+  displayText,
+  copyText,
+  size,
+}: {
+  displayText: string;
+  copyText: string;
+  size: number;
+}) {
+  const [textCopied, setTextCopied] = useState<boolean>(false);
+
+  async function handleCopy() {
+    setTextCopied(true);
+    if (typeof copyText == 'string') {
```

Comment:
copy test is requried to be a string based on the prop type - is this check necessary?

Reply:
apparently not - cant remember why I put it there. I think maybe it was throwing an error on JSX elements or something. Removed it.